### PR TITLE
docs(README): fix webpack 2 compatibility

### PR DESCRIPTION
**Pull request by simon04**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # yaml-frontmatter-loader for webpack
 
-YAML Frontmatter loader for [webpack](http://webpack.github.io/). Converts YAML in files to JSON. You should chain it with [json-loader](https://github.com/webpack/json-loader).
+YAML Frontmatter loader for [webpack](https://webpack.js.org/). Converts YAML in files to JSON. You should chain it with [json-loader](https://github.com/webpack/json-loader).
 
 ## Installation
 
 `npm install yaml-frontmatter-loader`
 
 ## Usage
```
Comment (michael-ciniawsky) [inline README.md]: `<h2 align="center"><a href="https://webpack.js.org/concepts/loaders/">Usage</a></h2>`

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # yaml-frontmatter-loader for webpack
```
Comment (michael-ciniawsky) [inline README.md]:
```
[![npm][npm]][npm-url]
[![node][node]][node-url]
[![deps][deps]][deps-url]
[![tests][tests]][tests-url]
[![coverage][cover]][cover-url]
[![chat][chat]][chat-url]

<div align="center">
  <img width="200" height="200"
    src="<!--  YAML Logo  or delete ;) -->">
  <a href="https://github.com/webpack/webpack">
    <img width="200" height="200"
      src="https://webpack.js.org/assets/icon-square-big.svg">
  </a>
  <h1>YAML Formatter Loader</h1>
</div>
```

File: README.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 ## Installation
 
 `npm install yaml-frontmatter-loader`
 
 ## Usage
 
-[Documentation: Using loaders](http://webpack.github.io/docs/using-loaders.html)
+[Documentation: Using loaders](https://webpack.js.org/concepts/loaders/)
 
 ``` javascript
```
Comment (michael-ciniawsky) [inline README.md]: **webpack.config.js**
\```javascript => \```js

```js
module.exports = {
  module: {
    rules: [
      {
         test: /\.md$/,
         use: [ 'json-loader', 'yaml-fontmatter-loader' ]
      }
    ]
  }
}
```

File: README.md [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+YAML Frontmatter loader for [webpack](https://webpack.js.org/). Converts YAML in files to JSON. You should chain it with [json-loader](https://github.com/webpack/json-loader).
 
 ## Installation
 
 `npm install yaml-frontmatter-loader`
 
 ## Usage
 
-[Documentation: Using loaders](http://webpack.github.io/docs/using-loaders.html)
+[Documentation: Using loaders](https://webpack.js.org/concepts/loaders/)
```
Comment (michael-ciniawsky) [inline README.md]: **file.md**
```md
### Example File
```

File: README.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
 
 ``` javascript
 
-var json = require("json!yaml-frontmatter-loader!./file.md");
+var json = require("json-loader!yaml-frontmatter-loader!./file.md");
 // => returns file.md as javascript object
 ```
```
Comment (michael-ciniawsky) [inline README.md]:
```diff
- ## LICENSE
```

```
[npm]: https://img.shields.io/npm/v/yaml-fontmatter-loader.svg
[npm-url]: https://npmjs.com/package/yaml-fontmatter-loader

[node]: https://img.shields.io/node/v/yaml-fontmatter-loader.svg
[node-url]: https://nodejs.org

[deps]: https://david-dm.org/webpack-contrib/yaml-fontmatter-loader.svg
[deps-url]: https://david-dm.org/webpack-contrib/yaml-fontmatter-loader

[tests]: http://img.shields.io/travis/webpack-contrib/yaml-fontmatter-loader.svg
[tests-url]: https://travis-ci.org/webpack-contrib/yaml-fontmatter-loader

[cover]: https://codecov.io/gh/webpack-contrib/yaml-fontmatter-loader/branch/master/graph/badge.svg
[cover-url]: https://codecov.io/gh/webpack-contrib/yaml-fontmatter-loader

[chat]: https://badges.gitter.im/webpack/webpack.svg
[chat-url]: https://gitter.im/webpack/webpack
```

File: README.md [hunk 6]
```diff
@@ -1,18 +1,18 @@
 # yaml-frontmatter-loader for webpack
 
-YAML Frontmatter loader for [webpack](http://webpack.github.io/). Converts YAML in files to JSON. You should chain it with [json-loader](https://github.com/webpack/json-loader).
+YAML Frontmatter loader for [webpack](https://webpack.js.org/). Converts YAML in files to JSON. You should chain it with [json-loader](https://github.com/webpack/json-loader).
 
 ## Installation
```
Comment (michael-ciniawsky) [inline README.md]: `Installation` => `Install`

**Review by michael-ciniawsky — CHANGES_REQUESTED** [review]

**Comment by simon04** [comment]
Thanks for your comments. The many changes were rather confusing to carry over, though. I hope to not have missed anything.

**Review by michael-ciniawsky — APPROVED** [review]
Thx 👍  very much 😛

**Pull request merged**